packages/edit-site/src/components/secondary-sidebar/style.scss

```diff
@@ -0,0 +1,49 @@
+.edit-site-editor__inserter-panel {
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+}
+
+.edit-site-editor__inserter-panel-header {
+	padding-top: $grid-unit-10;
+	padding-right: $grid-unit-10;
+	display: flex;
+	justify-content: flex-end;
+
+	@include break-medium() {
+		display: none;
+	}
+}
+
+.edit-site-editor__inserter-panel-content {
+	// Leave space for the close button
+	height: calc(100% - #{$button-size} - #{$grid-unit-10});
+
+	@include break-medium() {
+		height: 100%;
+	}
+}
+
+.edit-site-editor__list-view-panel {
```

Comment:
`NB:`

Maybe we could consolidate some of the duplicated styles for `.edit-site-editor__inserter-panel-content`, `.edit-site-editor__list-view-panel-content`, `.edit-site-editor__inserter-panel`, and `.edit-site-editor__list-view-panel`?